test/unit/toBuffer.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+// const fs = require('fs');
```

Comment:
We can remove this commented-out import.

Reply:
Yup, it's done. A leftover of the duplication of the file :)